working_pose:
  z: 0.3
  roll: 0
  pitch: 1.5708

use_numerical_ik: False

calibrate_location:
  x: 0.14
  y: -0.24

pickup_area:
  rects_x: 2
  rects_y: 8
  offset_x: 0.22
  offset_y: -0.5
  length_x: 0.25
  length_y: 1.0

scale_location:
  x: 0.14
  y: 0.3
  z: 0.085
  
cam_location:
  x: 0.14
  y: 0.3
  z: 0.085

discard_area:
  offset_x: 0.09
  offset_y: -0.35
  length_x: 0.08
  length_y: 0.13

